src/demo/app/app.component.ts

```diff
@@ -10,6 +10,7 @@ import { Component } from '@angular/core';
       </prx-navuser>
     </prx-header>
     <prx-modal></prx-modal>
+    <prx-overlay-spinner></prx-overlay-spinner>
```

Comment:
Since the overlay gets position: fixed anyways, does it matter if it's in the top-level component or not?